### PR TITLE
Enh/introduce slide action

### PR DESCRIPTION
**Pull request by PeterNgTr**
Added test

### CODE REVIEW

**Comment by PeterNgTr** [comment]
@DavertMik Added test

File: docs/helpers/Puppeteer.md [hunk 1]
```diff
@@ -466,6 +466,21 @@ I.dragAndDrop('#dragHandle', '#container');
 -   `srcElement`  located by CSS|XPath|strict locator.
 -   `destElement`  located by CSS|XPath|strict locator.
 
+### dragSlider
+
+Drag the srubber of a slider to a given position
```
Comment (pablopaul) [inline docs/helpers/Puppeteer.md]: @PeterNgTr "srubber" should be "scrubber", or?

File: docs/webapi/dragSlider.mustache [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Drag the srubber of a slider to a given position
```
Comment (pablopaul) [inline docs/webapi/dragSlider.mustache]: @PeterNgTr "srubber" should be "scrubber", or?

File: lib/helper/Puppeteer.js [hunk 3]
```diff
@@ -1299,6 +1299,27 @@ class Puppeteer extends Helper {
...
   }
 
+  /**
+   * {{> ../webapi/dragSlider }}
+   */
+  async dragSlider(locator, offsetX = 0) {
+    const src = await this._locate(locator);
+    assertElementExists(src, locator, 'Slider Element');
+
+    // Note: Using private api ._clickablePoint becaues the .BoundingBox does not take into account iframe offsets!
```
Comment (pablopaul) [inline lib/helper/Puppeteer.js]: @PeterNgTr "because" should be "because", or?

File: docs/webapi/dragSlider.mustache (hunk 2 above)
Comment (PeterNgTr) [reply docs/webapi/dragSlider.mustache]: fixed

File: lib/helper/Puppeteer.js (hunk 3 above)
Comment (PeterNgTr) [reply lib/helper/Puppeteer.js]: fixed

**Comment by DavertMik** [comment]
Thanks! Merging and releasing

**Pull request merged**